### PR TITLE
214605 apply mix format

### PR DESCRIPTION
**Pull request by aMasakiTakahashi**
https://acsmine.tok.access-company.com/redmine/issues/214605#note-14

### CODE REVIEW

File: test/auth_test.exs [hunk 1]
```diff
@@ -7,63 +7,97 @@ defmodule AntikytheraAws.Auth.InstanceProfileCredentialsTest do
...
+  @role_endpoint "http://169.254.169.254/latest/meta-data/iam/security-credentials/"
+  @test_role "test_role"
+  @creds_endpoint @role_endpoint <> @test_role
+  @creds_endpoint_success {:ok,
+                           %Res{
+                             body:
+                               ~S({"Code"  : "Success", "LastUpdated" : "2016-01-01T00:00:00Z",
                                                "Token" : "token",   "Expiration"  : "2099-12-31T23:59:59Z",
                                                "Type"  : "AWS-HMAC","AccessKeyId" : "aws_access_key_id",
                                                "SecretAccessKey" : "aws_secret_access_key"}),
```
Comment (irisTa56) [inline test/auth_test.exs]: Can we align here?

```suggestion
                               ~S({"Code"  : "Success", "LastUpdated" : "2016-01-01T00:00:00Z",
                                   "Token" : "token",   "Expiration"  : "2099-12-31T23:59:59Z",
                                   "Type"  : "AWS-HMAC","AccessKeyId" : "aws_access_key_id",
                                   "SecretAccessKey" : "aws_secret_access_key"}),
```

**Review by irisTa56 — APPROVED** [review]
I left a comment but LGTM overall.

**Review by aYukiSekiguchi — APPROVED** [review]
LGTM

File: test/auth_test.exs (hunk 1 above)
Comment (aMasakiTakahashi) [reply test/auth_test.exs]: I could not find a good way to align each line...
A newline is needed after the `~S(` to align each line, but in this case, the `body` will contain an extra newline.
Adding a newline elsewhere will either cause a syntax error or cause each line to not be aligned.

Comment (irisTa56) [reply test/auth_test.exs]: Sorry, including L22 might be misleading.
My suggestion is to delete preceding whitespaces of L23, 24,25, and to leave L22 as is.
(this change aligns starts of `"Code"`, `"Token"`, `"Type"`, `"SecretAccessKey"`)
In my local environment, this seems to be possible.

Comment (aMasakiTakahashi) [reply test/auth_test.exs]: Thank you for your explanation.
I fixed in 5651de2
Is this what you intended?

Comment (irisTa56) [reply test/auth_test.exs]: Yes. Thanks!

**Review by Fumipo-Theta — COMMENTED** [review]
LGTM !

**Pull request merged**